wordpress-mobile/WordPress-iOS
# [Gutenberg] Implement aztec attachment delegate

This PR implement the changes introduced by https://github.com/wordpress-mobile/gutenberg-mobile/pull/281 , and implements `AztecAttachmentDelegate` in `GutenbergViewController`.

The main objective of this PR is to avoid a crash when an instance of Aztec try to load an image. It's not common scenario for Alpha but it might happen.

**To test:**

- Run `pod install`
- Clean build folder.
- Build and run the app in Gutenberg mode.
- Start a new post (+ button in the middle of the tab bar)
- Switch to HTML mode and paste this snippet:
```html

<p>Hi there</p>
<img src="https://etoledomatomicsite01.blog/wp-content/uploads/2018/04/kitty-cat-kitten-pet-45201.jpg" alt="" class="wp-image-43">

```
- Switch back to blocks view.
- Check that it shows the placeholder image.
  - If it shows the image directly, it means that the image was cached.
  - If you want you can try again with another image url. 
- Check that the image appears when the placeholder disappears.
  - It might happen that the loaded image won't appear after the placeholder.
  - If this happen, switch to html view and back and the image should be there.
  - This issue seems to be non-trivial and is not really important for Alpha, so I'd prefer get back to it later on.

![simulator screen shot - iphone 8 - 2018-11-27 at 19 21 49](https://user-images.githubusercontent.com/9772967/49115574-edd62880-f279-11e8-8fbc-3fb44c379f35.png)
